Extension/src/LanguageServer/extension.ts

```diff
@@ -452,6 +452,7 @@ function realActivation(): void {
 
     disposables.push(vscode.workspace.onDidChangeConfiguration(onDidChangeSettings));
     disposables.push(vscode.window.onDidChangeActiveTextEditor(onDidChangeActiveTextEditor));
+    ui.activeDocumentChanged(); // Handle already active documents (for non-cpp files that we don't register didOpen).
```

Comment:
This was also causing a bug where the config wasn't being shown initially for open documents.